src/lib/i18n/texts/de.json

```diff
@@ -66,22 +66,22 @@
       "losses": "Niederlagen",
```

Comment:
there definitely is plans to, right now the embed page was just made as fast as possible. i need to update it to support translations (basically bring it into the main site)